### PR TITLE
feat(24.04): add libyaml-0-2 slice

### PR DESCRIPTION
**Pull request by endersonmaia**
# Proposed changes

Add libyaml-0-2 slice

## Testing

```bash
./chisel cut --release ./  --root rootfs  --arch=amd64 libyaml-0-2_libs
2024/06/24 15:34:54 Processing ./ release...
2024/06/24 15:34:54 Selecting slices...
2024/06/24 15:34:54 Fetching ubuntu 24.04 noble suite details...
2024/06/24 15:34:55 Release date: Thu, 25 Apr 2024 15:10:33 UTC
2024/06/24 15:34:55 Fetching index for ubuntu 24.04 noble main component...
2024/06/24 15:34:55 Fetching index for ubuntu 24.04 noble universe component...
2024/06/24 15:34:55 Fetching ubuntu 24.04 noble-security suite details...
2024/06/24 15:34:55 Release date: Mon, 24 Jun 2024 11:09:13 UTC
2024/06/24 15:34:55 Fetching index for ubuntu 24.04 noble-security main component...
2024/06/24 15:34:55 Fetching index for ubuntu 24.04 noble-security universe component...
2024/06/24 15:34:55 Fetching ubuntu 24.04 noble-updates suite details...
2024/06/24 15:34:55 Release date: Mon, 24 Jun 2024 13:56:55 UTC
2024/06/24 15:34:55 Fetching index for ubuntu 24.04 noble-updates main component...
2024/06/24 15:34:55 Fetching index for ubuntu 24.04 noble-updates universe component...
2024/06/24 15:34:55 Fetching pool/main/g/gcc-14/gcc-14-base_14-20240412-0ubuntu1_amd64.deb...
2024/06/24 15:34:55 Fetching pool/main/g/glibc/libc6_2.39-0ubuntu8.2_amd64.deb...
2024/06/24 15:34:55 Fetching pool/main/g/gcc-14/libgcc-s1_14-20240412-0ubuntu1_amd64.deb...
2024/06/24 15:34:55 Fetching pool/main/liby/libyaml/libyaml-0-2_0.2.5-1build1_amd64.deb...
2024/06/24 15:34:55 Extracting files from package "gcc-14-base"...
2024/06/24 15:34:55 Extracting files from package "libc6"...
2024/06/24 15:34:55 Extracting files from package "libgcc-s1"...
2024/06/24 15:34:55 Extracting files from package "libyaml-0-2"...

```

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

File: slices/libyaml-0-2.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+essential:
+  - libyaml-0-2_copyright
+
+slices:
+  libs:
+    essential:
+      - gcc-14-base_copyright
+      - libc6_libs
+      - libgcc-s1_libs
```
Comment (zhijie-yang) [inline slices/libyaml-0-2.yaml:11]: From what I see, `libc6` is the only direct dependency of `libyaml-0-2`. Could you briefly explain the reason to include `libgcc-s1` and `gcc-14-base`?

**Review by zhijie-yang — CHANGES_REQUESTED** [review]

File: slices/libyaml-0-2.yaml (hunk 1 above)
Comment (endersonmaia) [reply slices/libyaml-0-2.yaml:11]: I used the following command to look for dependencies:

```
# apt-cache depends --recurse --no-recommends --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances libyaml-0-2 | grep "^\w" | sort -u
gcc-14-base
libc6
libgcc-s1
libyaml-0-2
```

But I think it was too much.

I'll send a fixup.

**Review by zhijie-yang — APPROVED** [review]
LGTM, thanks!

**Review by cjdcordeiro — APPROVED** [review]
short and sweet! thanks!

**Pull request merged**